pages/signedout.js

```diff
@@ -0,0 +1,52 @@
+import { useEffect } from 'react'
```

Comment:
Can you double check this page meets accessibility?

Reply:
Tested with screen reader and added H1